src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptDebugDataTipInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal struct VSTypeScriptDebugDataTipInfo
```

Comment:
```suggestion
    internal readonly struct VSTypeScriptDebugDataTipInfoWrapper
```

Reply:
Why do we need `-Wrapper` suffix when we already have `VSTypeScript-` prefix. Seems like unnecessary clutter.